Userland/Applications/FileManager/PropertiesWindow.cpp

```diff
@@ -110,6 +110,15 @@ PropertiesWindow::PropertiesWindow(const String& path, bool disable_rename, Wind
         general_tab.remove_child(*link_location_widget);
     }
 
+    auto bitmap = Gfx::Bitmap::try_load_from_file(path);
```

Comment:
Ideally we'd avoid _actually_ loading the image, but I don't think we have the means to get just metadata about an image right now...

Reply:
Actually, that's exactly what `Gfx::ImageDecoder::size()` does :^)

Reply:
Updated change to use it then ^^